src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java

```diff
@@ -91,8 +87,8 @@ private static int getSystemProperty(String name, String defaultValue) {
 			final ProxyData proxyData = detectProxyServer(requestModel.getUrl());
 			if (proxyData != null) {
 				if (log.isDebugEnabled()) {
-					log.debug(String.format("Using proxy @" + proxyData.host
```

Comment:
this `String.format(` did nothing, as there were no varargs for stuff to be formatted.